pipelines/curate_identity.py

```
@@ -0,0 +1,547 @@
+        return None
+    value = claimed.strip().lower()
+    if value in CANONICAL_PROVENANCE:
+        return value
+    # Preserve the repository's documented first-match order.  A synthetic
+    # narrative claiming live/production use remains designed even if it also
+    # mentions simulated or HIL calibration.
+    if (
+        value.startswith("real")
+        or value.startswith("live")
+        or "actions live" in value
+        or "production" in value
```

Ruff (0.16.1)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)